docs/apps/lnw/es2k/es2k-lnw-overlay-vms.md

```diff
@@ -45,7 +45,7 @@ System under test will have above topology running the networking recipe. Link P
 
 ## Creating the topology
 
-Follow steps mentioned in [Running Infrap4d on Intel E2100](/guides/es2k/running-infrap4d.md) for starting `infrap4d` process and creating protobuf binary for `fxp-net_linux-networking-v2` p4 program.
+Follow steps mentioned in [Running Infrap4d on Intel E2100](/docs/guides/es2k/running-infrap4d.md) for starting `infrap4d` process and creating protobuf binary for `fxp-net_linux-networking-v2` p4 program. Modify `program-name` in `/usr/share/stratum/es2k/es2k_skip_p4.conf` to `fxp-net_linux-networking-v2` before exploring the usecase.
```

Comment:
```suggestion
Follow steps mentioned in [Running Infrap4d on Intel E2100](/guides/es2k/running-infrap4d.md) for starting `infrap4d` process and creating protobuf binary for `fxp-net_linux-networking-v2` p4 program. Modify `program-name` in `/usr/share/stratum/es2k/es2k_skip_p4.conf` to `fxp-net_linux-networking-v2` before exploring the usecase.
```
Relative links in Sphinx documents treat `/docs` as the implied root. You must omit **/docs** or the link will be broken in the user guide. Links do not operate the same when you're browsing the .md files as they do in the rendered document.